docs/source/development/unit_testing.ipynb

```diff
@@ -77,82 +77,75 @@
-    "| Method                      | Checks that                |\n",
-    "|:----------------------------|:---------------------------|\n",
-    "| `assertEqual(a, b)`         | `a == b`                   |\n",
-    "| `assertNotEqual(a, b)`      | `a != b`                   |\n",
-    "| `assertTrue(x)`             | `bool(x) is True`          |\n",
-    "| `assertFalse(x)`            | `bool(x) is False`         |\n",
-    "| `assertIn(a, b)`            | `a in b`                   |\n",
-    "| `assertNotIn(a, b)`         | `a not in b`               |\n",
-    "| `assertIsInstance(a, b)`    | `isinstance(a, b)`         |\n",
-    "| `assertNotIsInstance(a, b)` | `not isinstance(a, b)`     |\n",
-    "| `assertRaises(exc)`         | exception `exc` is raised. |\n",
-    "\n",
-    "For a complete list see the [unittest.TestCase documentation](https://docs.python.org/3/library/unittest.html#unittest.TestCase)."
+    "Often we want to run a test for different combinations of\n",
+    "parameters. You can do so by parametrizing tests via a decorator."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [],
+   "source": [
+    "import pytest\n",
+    "\n",
+    "@pytest.mark.parametrize(\"x,y\", [(3, 5), (-1, 1), (0, 1.2)])\n",
+    "def test_symmetric(x, y):\n",
+    "    assert add(x, y) == add(y, x)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "### NumPy Assertions\n",
+    "Read more about test parametrization and fixtures [here](https://docs.pytest\n",
+    ".org/en/stable/parametrize.html).\n",
     "\n",
-    "Often assertions which compare arrays are needed for tests. The class `tests.testing.NumpyAssertions` wraps assert statements used in NumPy, e.g. \n",
+    "### NumPy Assertions\n",
     "\n",
-    "| Method                                             | Checks that                                                 |\n",
-    "|:---------------------------------------------------|:------------------------------------------------------------|\n",
-    "| `assertApproxEqual(a, b, significant=7)`           | scalars are equal up to significant digits.                 |\n",
-    "| `assertAllClose(a, b, rtol, atol, equal_nan=True)` | arrays are the same within a desired tolerance.             |\n",
-    "| `assertArrayEqual(a, b)`                           | arrays have the same shape and elements.                    |\n",
-    "| `assertArrayLess(a, b)`                            | all elements of `a` are strictly smaller than those of `b`. |"
-   ]
+    "Often assertions which compare arrays are needed for tests. The module [numpy\n",
+    ".testing](https://numpy.org/doc/stable/reference/routines.testing.html) offers\n",
+    "support for commonly used tests involving numeric arrays, such as\n",
+    "comparison of all elements up to a certain tolerance."
+   ],
```

Comment:
I suppose this block was much more important in the unittest context. In pytest, this info may not be needed afterall? I dont know, your decision :)